### PR TITLE
Login File created

### PR DESCRIPTION
**Pull request by ilianafantano**
because i want so

### CODE REVIEW

File: src/test/java/com/automation/loginTest.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.automation;
+
+public class loginTest {
```
Comment (chiragkhimani) [inline src/test/java/com/automation/loginTest.java]: @ilianafantano - Can you please make class name Capital?

Comment (ilianafantano) [reply src/test/java/com/automation/loginTest.java]: no problem boss

**Comment by Betterat70sog** [comment]
@ilianafantano how are you ?

**Comment by chiragkhimani** [comment]
Good job!!

**Comment by ilianafantano** [comment]
> @ilianafantano how are you ?

normalyok

**Review by chiragkhimani — APPROVED** [review]

**Pull request merged**